main.js

```diff
@@ -0,0 +1,186 @@
+const getquizNum = document.getElementById('quizNum');
+const getcategory = document.getElementById('category');
+const getdifficulty = document.getElementById('difficulty');
+const getquestion = document.getElementById('question');
+const getanswers = document.getElementById('answers').children;
+
+// スタート画面表示の関数
+const getStart = () => {
+  getquizNum.innerHTML = 'ようこそ';
+  getcategory.innerHTML = '';
+  getdifficulty.innerText = '';
+  getquestion.innerHTML = '以下のボタンをクリック';
+  getanswers[0].innerHTML = '開始';
+  getanswers[1].style.display = 'none';
+  getanswers[2].style.display = 'none';
+  getanswers[3].style.display = 'none';
+
+  document.getElementById('startClick').addEventListener('click', (e) => {
+    document.getElementById('startClick').removeAttribute('id');
+    getanswers[0].style.display = 'none';
+    waitApi();
+    getQuizArray();
+  });
```

Comment:
`getQuizArray` が `async` なので、呼び出す側は以下のようになっている必要があります。
（今回は問題なく動いていますが、推奨されないです）

```suggestion
  document.getElementById('startClick').addEventListener('click', async (e) => {
    document.getElementById('startClick').removeAttribute('id');
    getanswers[0].style.display = 'none';
    waitApi();
    await getQuizArray();
  });
```